config/default.yaml

```diff
@@ -1,24 +1,27 @@
 function:
-  study_mapping: False
-  check_for_missing_modules: True
+  study_mapping: True
+  check_for_missing_modules: False
   plot_lpGBTLoads: False
   plot_ModuleLoads: False               
 
-print_level: 0
+print_level: 2
 output_dir: .
 
 study_mapping:
   MappingFile: data/FeMappingV7.txt
-  CMSSW_ModuleHists: data/ROverZHistograms_v11-NG-1.root
+  CMSSW_ModuleHists: data/ROverZHistograms2D_20200526.root
   #algorithm: save_root
   algorithm: random_hill_climb
   # algorithm: simulated_annealing
   # algorithm: genetic_alg      
   #initial_state: bestsofar
   initial_state: random
-  #initial_state: bestsofar_38.npy       
+  #initial_state: bundles_job_default.npy   
   random_seed: ~
   max_iterations: 100000 
+  #minigroup_type: 'bylayer_silicon_seprated'
+  #minigroup_type: 'bylayer'
+  minigroup_type: 'minimal'
```

Comment:
I would add these lines (minigroup_type), but remove the other changes to this file.